client/app/helper/controller-helper.service.js

```diff
@@ -60,6 +61,10 @@ class ControllerHelper {
         }
         return "";
     }
+
+    scrollPageToTop () {
+        this.$timeout(() => scrollTo(0, 0), 100);
```

Comment:
We will need a timeout here to avoid scroll position change due to DOM manipulation. For example, when we click on a menuitem from a menu and have the scrollToTop being called there, it resets the scroll back to the position of the menu after the menu disappears. We will need a timeout until we have the $event being passed into the on-click of the menu item, so that we can stop the propagation.